<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopWidget</class>
 <widget class="QWidget" name="ShopWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShopWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="itemBox">
     <property name="title">
      <string>Item</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="itemRandom">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Random</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="equipmentLayout">
     <item>
      <widget class="QGroupBox" name="weaponBox">
       <property name="title">
        <string>Weapon</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="weaponRandom">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="armorBox">
       <property name="title">
        <string>Armor</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="armorRandom">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="otherBox">
       <property name="title">
        <string>Other</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="otherRandom">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="eternalCorridorUnlockAllBox">
     <property name="title">
      <string>Eternal Corridor Unlock All</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="eternalCorridorUnlockAllEnable">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
